src/ui/heatmap/mapChooseMetaverse.ui.tsx

```diff
@@ -10,15 +10,15 @@ interface MapChooseMetaverseUIProps {
   setSelectCoord: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
-export default function MapChooseMetaverseUI({metaverse, setMetaverse, selectMetaverse, setSelectMetaverse, setSelectfilter, setSelectCoord}: MapChooseMetaverseUIProps) {
+export default function MapChooseMetaverseUI({ metaverse, setMetaverse, selectMetaverse, setSelectMetaverse, setSelectfilter, setSelectCoord }: MapChooseMetaverseUIProps) {
 
   return (
     <div className="relative">
-      <button onClick={() => {setSelectMetaverse(!selectMetaverse); setSelectfilter(false); setSelectCoord(false);}}>
+      <button onClick={() => { setSelectMetaverse(!selectMetaverse); setSelectfilter(false); setSelectCoord(false); }}>
```

Comment:
don't call multiple functions on one line, instead create a function for call multiple functions